shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -706,3 +701,56 @@ expectedStEx7 =
 
 ex7 :: CHAINExample
 ex7 = CHAINExample (Slot 215) expectedStEx6 blockEx7 expectedStEx7
+
+
+--  | Example 8 - create the first non-trivial reward update by processing an
+--  empty block late in the epoch.
+
+
+blockEx8 :: Block
```

Comment:
apologies, this next example (#8) is a WIP and was not supposed to be a part of the code review changes.